WcaOnRails/app/views/registrations/psych_sheet_event.html.erb

```diff
@@ -30,19 +30,19 @@
           <td class="pos <%= registration.tied_previous ? "tied-previous" : "" %>">
             <%= registration.pos %>
           </td>
-          <td class="name"><%= registration.name %></td>
+          <td class="name"><%= registration.select_name %></td>
```

Comment:
@jonatanklosko I started with wca_id but the `Registration` model has a method `wca_id` and a method `name`. It seems that in the view when I use `registration.name` it calls the method. That's why I used a different identifier for these.

These methods check if a registration has a user, and if so uses the value from user, otherwise it gets it from Preregs. I think these methods can all go when #275 is done.